psi4/run_psi4.py

```diff
@@ -212,10 +219,6 @@ def custom_formatwarning(msg, *args, **kwargs):
 sys.path.insert(1, lib_dir)
 import psi4  # isort:skip
```

Comment:
If we want to short-circuit the `import psi4`, shouldn't we remove the `import psi4`?

Reply:
that's later, after the `args["version"]:` call has already `sys.exit()`ed.